binding/nodejs/index.js

```diff
@@ -116,22 +133,56 @@ class Porcupine {
     }
 
     const packed = pvPorcupine.init(
-        accessKey, 
-        modelPath, 
-        keywordPaths.length, 
-        keywordPaths, 
-        sensitivities);
+      accessKey,
+      modelPath,
+      keywordPaths.length,
+      keywordPaths,
+      sensitivities
+    );
     const status = Number(packed % 10n);
     if (status !== PV_STATUS_T.SUCCESS) {
       pvStatusToException(status, "Porcupine failed to initialize");
     }
     this.handle = packed / 10n;
 
-    this.frameLength = pvPorcupine.frame_length();
-    this.sampleRate = pvPorcupine.sample_rate();
-    this.version = pvPorcupine.version();
+    this._frameLength = pvPorcupine.frame_length();
+    this._sampleRate = pvPorcupine.sample_rate();
+    this._version = pvPorcupine.version();
+  }
+
+  /**
+   * @returns number of audio samples per frame (i.e. the length of the array provided to the process function)
+   * @see {@link process}
+   */
+  get frameLength() {
+    return this._frameLength;
+  }
+
+  /**
+   * @returns the audio sampling rate accepted by Porcupine
+   */
+  get sampleRate() {
+    return this._sampleRate;
+  }
+
+  /**
+   * @returns the version of the Porcupine engine
+   */
+  get version() {
+    return this._version;
   }
 
+  /**
+   * Process a frame of pcm audio.
+   *
+   * @param {Array} frame 16-bit integers of 16kHz linear PCM mono audio.
```

Comment:
We shouldn't mention sample rate since that's a variable.
```suggestion
   * @param {Array} frame of mono, 16-bit, linear-encoded PCM audio.
```